src/components/services/pathresolver/jsonforms-pathresolver.ts

```diff
@@ -1,17 +1,17 @@
 
 import {PathUtil} from '../pathutil';
 
-export class PathResolver  {
+export class PointerResolver  {
```

Comment:
PointerResolver sounds like C++ ;)

Maybe use `RefResolver` or `ReferenceResolver` since this is how they are called within the models.